### PR TITLE
e2e: yet more cleanup of BeTrue/BeFalse

### PR DESCRIPTION
**Pull request by edsantiago**
Thanks to Paul for teaching me about HaveKey()

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
This is probably the last one I'll do. There are still a small number of `BeTrue()`s left, and two `BeFalse()`s, but all the remaining ones are hard to fix.

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by edsantiago** [comment]
/hold

Found a bug (not mine); but will wait for CI to pass/fail before pushing fix

File: test/e2e/prune_test.go [hunk 1]
```diff
@@ -189,11 +187,11 @@ var _ = Describe("Podman prune", func() {
 
 		after := podmanTest.Podman([]string{"images", "-a"})
 		after.WaitWithDefaultTimeout()
-		Expect(none).Should(Exit(0))
-		hasNoneAfter, result := none.GrepString("<none>")
```
Comment (edsantiago) [inline test/e2e/prune_test.go:193]: I'm pretty sure this was a copy-paste bug from way back.

**Pull request merged**